src/_layouts/default.html

```diff
@@ -2,9 +2,9 @@
 <html lang="en">
 	{% include head.html %}
 	{% include header.html %}
-	<body class="font-weight-normal ff-roboto bg-white pt-5">
+	<body class="font-weight-normal bg-white pt-5">
 		{% include navbar.html %}
-		<div class="container" style="max-width: 1440px;">
+		<div class="container" style="max-width: 1440px; margin-bottom: 80px;">
```

Comment:
Shouldn't we have this in CSS instead on the html elements?

Reply:
Now they are in CSS also I may remove the width because Bootstrap should handle it but somehow container does not go wide enough so I did kinda hack it, however, I will fix this later.